images/image-repo-list-2004

```diff
@@ -1,5 +1,6 @@
 e2eRegistry: k8sprow.azurecr.io/kubernetes-e2e-test-images
 gcAuthenticatedRegistry: e2eprivate
+promoterE2eRegistry: k8sprow.azurecr.io/kubernetes-e2e-test-images
```

Comment:
This update make this file the same as the `master` list.  Can we just get rid of the diffs for 2004?

Reply:
There is one slight difference between these two files:
```diff
diff --git a/images/image-repo-list-master b/images/image-repo-list-2004
index 7c27d06..b672005 100644
--- a/images/image-repo-list-master
+++ b/images/image-repo-list-2004
@@ -1,4 +1,5 @@
 e2eRegistry: k8sprow.azurecr.io/kubernetes-e2e-test-images
+gcAuthenticatedRegistry: e2eprivate
 promoterE2eRegistry: k8sprow.azurecr.io/kubernetes-e2e-test-images
 etcdRegistry: k8sprow.azurecr.io/kubernetes-e2e-test-images
 gcEtcdRegistry: k8sprow.azurecr.io/kubernetes-e2e-test-images
```
Reason for this additional field: https://kubernetes.slack.com/archives/C0SJ4AFB7/p1614105191048500?thread_ts=1614101956.044700&cid=C0SJ4AFB7

Reply:
Maybe we should make those explicit in this file?  so it's easier to see what all the required entries are and were they are currently pointing?